src/simulations/local_elec.py

```diff
@@ -559,14 +557,7 @@ def policy_to_rewardvar(
             bool_penalty, date, loads, E_req_only, h, last_step, home_vars)
 
         if self.prm['grd']['manage_voltage']:
-            start = time.time()
             hourly_line_losses, voltage = self.network.pf_simulation(home_vars['netp'])
-            end = time.time()
-            voltage_squared = np.square(voltage)
```

Comment:
did you mean to delete this?
`
voltage_squared = np.square(voltage)
`

Reply:
uh sorry, this was a mistake and the reason why the test fails. I'll correct this asap, thanks.